catalog/USGS/USGS_NLCD_RELEASES_2019_REL_NALCMS.jsonnet

```diff
@@ -0,0 +1,269 @@
+local id = 'USGS/NLCD_RELEASES/2019_REL/NALCMS';
+local subdir = 'USGS';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+
+local license = spdx.cc0_1_0;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  // TODO(schwehr): Remove skip_indexing when the preview image is ready.
+  'gee:skip_indexing': true,
+  'gee:user_uploaded': true,
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  // TODO(schwehr): help with title, version, and column width in description please
```

Comment:
For the column width, you can copy the text from

https://github.com/google/earthengine-catalog/blob/3d23916f0d83aba2538c5dc9222a5e9ca8359cf6/catalog/USGS/USGS_NLCD_RELEASES_2019_REL_NALCMS.jsonnet